README.md

```diff
@@ -5,19 +5,28 @@ The Zig logo and mascots are licensed under the Attribution-ShareAlike 4.0 Inter
 
 ## Main Logo
 
-<img alt="Zig Logo" src="zig-logo.svg" width="300">
+<img alt="Zig Logo" src="full logo/zig_logo_black.svg" width="300">
 
-## Icon
+`full logo/zig_color_black.svg` is the preferred logo, containing both logomark and logotype.
 
-<img alt="Zig Icon" src="zig-icon.svg" width="300">
+## Logo mark
 
-zig-icon.svg is an icon of just the graphical part of the logo.
+<img alt="Zig Mark" src="logo mark/zig_mark_color.svg" width="300">
+
+`zig_mark_color.svg` is the logomark (icon) alone.
 
 ## Favicon
 
 <img alt="Zig Favicon" src="zig-favicon.png" width="16" height="16">
 
-zig-icon.png is a small version of the icon suitable for a website favicon.
+`zig-icon.png` is a small version of the icon made for website favicons.
+
+# Logo versions
+
+* `full logo/zig_color_black.svg` is the main logo, preferably used on applicable backgrounds.
+* `full logo/zig_color_white.svg` when visibility of the main logotype ("ZIG") is too low.
+* `full logo/zig_black.neg.svg` where color reproduction is not possible, or visibility of colored symbol is too low.
+* `full logo/zig_white.neg.svg` where color reproduction is not possible, or visibility of `zig_dark.neg.svg` is too low.
```

Comment:
the "full logo" part can be removed now, yeah?

Reply:
You mean the part of the reference in the README.md? Yeah, doesn't matter much as the naming is clear like it is now.

Aside from that, yes, it is ready :)